### DISCUSSION TITLE
How to stop supabase if config file is deleted.

### DISCUSSION THREAD

**Discussion by veeru004** [supabase/supabase#7240] [Questions]
I have deleted whole supabase folder and i forgot to run 'supabase stop' command. now supabase is running in background how to stop it?

**Comment by nattstack** [comment]
I believe this is related to Supabase CLI and you are running Supabase locally. May I get more context on your problem. How do you know Supabase is running in the background? If you exit or force quit your terminal, wouldn't that stop the Supabase locally?
